src/index.js

```diff
@@ -9,6 +9,7 @@ refractor.register(require('refractor/lang/json.js'));
 refractor.register(require('refractor/lang/typescript.js'));
 refractor.register(require('refractor/lang/nginx.js'));
 refractor.register(require('refractor/lang/ruby.js'));
+refractor.register(require('refractor/lang/scss.js'));
```

Comment:
Why is this one necessary now?

Reply:
I found there's usage here https://github.com/webpack/webpack.js.org/blob/9970f584a53dec88cdc196b30aef0be0ef74ba42/src/content/guides/entry-advanced.md#L32

![image](https://user-images.githubusercontent.com/1091472/91166593-2c29a180-e705-11ea-9499-031b43a170e9.png)

I can remove it if you want.

Reply:
I see, no, it is fine :)